src/Microsoft.IdentityModel.Xml/DsigSerializer.cs

```diff
@@ -73,6 +69,18 @@ public DSigSerializer()
         {
         }
 
+        /// <summary>
+        /// Gets or sets the maximum number of Transforms that are allowed on a Reference
```

Comment:
nit: missing a period at the end of the comment.

Reply:
thanks